packages/volto/news/7414.bugfix

```diff
@@ -0,0 +1 @@
+The cancel/clear "X" button inside the widget now includes an accessible label when active. @Wagner3UB
```

Comment:
Remember someone reading the release notes won't have any context. So be clear which widget you are talking about!
```suggestion
The cancel/clear "X" button inside the DatetimeWidget now includes an accessible label when active. @Wagner3UB
```